ipa.tf

```diff
@@ -98,7 +104,9 @@ app-edge:
 app-edge:
   alternateDomain: ""
   image:
-    registry: ${var.local_registry_enabled ? "local-registry.${local.dns_name}" : "harbor.devops.indico.io"}/indico
+    registry: ${var.local_registry_enabled ? "local-registry.${local.dns_name}" : "${var.local_registry_enabled ? "local-registry.${local.dns_name}" : "${var.image_registry}"}"}/indico
```

Comment:
Double check on var.local_registry_enabled, will be confusing if we look at this again in a few months

Reply:
pushed fix.